widget/embedded/src/components/NotificationContent/NotificationContent.tsx

```diff
@@ -0,0 +1,108 @@
+import type { Notification } from '../../types/notification';
+
+import {
+  ChainToken,
+  ChevronRightIcon,
+  ListItemButton,
+  Typography,
+} from '@rango-dev/ui';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { useMetaStore } from '../../store/meta';
+import { useNotificationStore } from '../../store/notification';
+
+import { Container, Images, List } from './NotificationContent.styles';
+
+export function NotificationContent() {
+  const MAX_NOTIFICATIONS_DISPLAYED = 4;
+  const navigate = useNavigate();
+  const notifications: Notification[] =
+    useNotificationStore.use.notifications();
+
+  const setAsRead = useNotificationStore.use.setAsRead();
+
+  const { tokens, blockchains } = useMetaStore.use.meta();
+  const sortedNotification = notifications
+    .sort((a, b) => {
+      const POSITIVE = 1;
+      const NEGATIVE = -1;
+      if (a.read === b.read) {
+        return a.creationTime < b.creationTime ? POSITIVE : NEGATIVE;
+      }
+      return a.read ? POSITIVE : NEGATIVE;
+    })
+    .slice(0, MAX_NOTIFICATIONS_DISPLAYED);
+
+  const handleOnClick = (requestId: Notification['requestId']) => {
+    setAsRead(requestId);
+    navigate(`/swaps/${requestId}`);
```

Comment:
![image](https://github.com/rango-exchange/rango-client/assets/120931880/025b4987-e70b-4bcf-9220-fad833aecf9a)
after reloading the page navigation doesn't work properly.
because we are using both router and memory router we cannot get the URL params in swap history item page.
please check the transaction history page